tests/test_linxura.py

```diff
@@ -0,0 +1,213 @@
+            "command_id": 10,
+        },
+    )
+
+    message = b"\x18\n\n\x02\x00\x19\x0f\x00\xfe\xff0\x01"
+    listener.reset_mock()
+    device.handle_message(260, cluster.cluster_id, 1, 1, message)
+    assert listener.zha_send_event.call_count == 1
+    assert listener.zha_send_event.call_args == mock.call(
+        "button_3_remote_button_double_press",
+        {
+            "button": "button_3",
+            "press_type": "remote_button_double_press",
+            "command_id": 10,
+        },
+    )
+
+    message = b"\x18\n\n\x02\x00\x19\x11\x00\xfe\xff0\x01"
+    listener.reset_mock()
+    device.handle_message(260, cluster.cluster_id, 1, 1, message)
+    assert listener.zha_send_event.call_count == 1
+    assert listener.zha_send_event.call_args == mock.call(
+        "button_3_remote_button_long_press",
+        {
+            "button": "button_3",
+            "press_type": "remote_button_long_press",
+            "command_id": 10,
+        },
+    )
+
+    message = b"\x18\n\n\x02\x00\x19\x13\x00\xfe\xff0\x01"
+    listener.reset_mock()
+    device.handle_message(260, cluster.cluster_id, 1, 1, message)
+    assert listener.zha_send_event.call_count == 1
+    assert listener.zha_send_event.call_args == mock.call(
+        "button_4_remote_button_short_press",
+        {
+            "button": "button_4",
+            "press_type": "remote_button_short_press",
+            "command_id": 10,
+        },
+    )
+
+    message = b"\x18\n\n\x02\x00\x19\x15\x00\xfe\xff0\x01"
+    listener.reset_mock()
+    device.handle_message(260, cluster.cluster_id, 1, 1, message)
+    assert listener.zha_send_event.call_count == 1
+    assert listener.zha_send_event.call_args == mock.call(
+        "button_4_remote_button_double_press",
+        {
+            "button": "button_4",
+            "press_type": "remote_button_double_press",
+            "command_id": 10,
+        },
+    )
+
+    message = b"\x18\n\n\x02\x00\x19\x17\x00\xfe\xff0\x01"
+    listener.reset_mock()
+    device.handle_message(260, cluster.cluster_id, 1, 1, message)
+    assert listener.zha_send_event.call_count == 1
+    assert listener.zha_send_event.call_args == mock.call(
+        "button_4_remote_button_long_press",
+        {
+            "button": "button_4",
+            "press_type": "remote_button_long_press",
+            "command_id": 10,
+        },
+    )
+
+
+@pytest.mark.parametrize("quirk", (zhaquirks.linxura.button.LinxuraButton,))
+async def test_edge_case_request(zigpy_device_from_quirk, quirk):
+    """Test edge case."""
+
+    device = zigpy_device_from_quirk(quirk)
+
+    # endpoint 1
+    cluster = device.endpoints[1].ias_zone
+    ias_zone_listener = ClusterListener(cluster)
+    ias_zone_status_attr_id = IasZone.AttributeDefs.zone_status.id
+
+    cluster.update_attribute(ias_zone_status_attr_id, 4)
+    assert (
+        len(ias_zone_listener.attribute_updates) == 1
+    )  # No update should occur for state >= 4
```

Comment:
What is the purpose of this test? The comment states "No update should occur for state >= 4", but the IAS zone status attribute update always happens, regardless of "state".

Reply:
Update: I've reworked the tests and removed this for now.